webbpsf/webbpsf_core.py

```diff
@@ -920,6 +921,87 @@ def load_was_opd(self, inputWasOpd, size=1024, save=False, filename='new_was_opd
 
         return fits.HDUList(fits.ImageHDU(newopd, header=hdr))
 
+    def psf_grid(self, all_detectors=True, num_psfs=16, single_psf_centered=True, use_detsampled_psf=False,
+                 save=False, outfile=None, overwrite=True, **kwargs):
+        """
+        Create a PSF library in the form of a grid of PSFs across the detector based on the specified instrument,
+        filter, and detector. The output file will contain a 3D array with axes [i, y, x] where i is the PSF position
+        on the detector grid and (y,x) is the 2D PSF.
+
+        Parameters
+        ----------
+        all_detectors : bool
+            If True, run all detectors for the instrument. If False, run for the detector set in
+            the instance. Default is True
+        use_detsampled_psf : bool
```

Comment:
This is tied back to that unresolved question in the telecon... But I would think this is different enough *conceptual* behavior that instead of being a keyword it should be two methods, one called, say, `epsf_grid`, and the other `psf_grid`.  They could still share most of their functionality in an internal private (leading `_`) method, but it would reinforce to the user that these are conceptually very different objects.